packages/transformers/vue/src/VueTransformer.js

```diff
@@ -337,6 +338,36 @@ ${
             }
             style.lang = extname(style.src);
           }
+          let toInstall;
+          switch (style.lang) {
+            case 'less':
+              toInstall = 'less';
+              break;
+            case 'stylus':
+            case 'styl':
+              toInstall = 'stylus';
+              break;
+            case 'scss':
+            case 'sass':
+              toInstall = 'sass';
+              break;
+            case 'css':
+            case undefined:
+              break;
+            default:
+              throw new ThrowableDiagnostic({
+                diagnostic: {
+                  message: `Unknown style language: "${style.lang}"`,
+                  origin: '@parcel/transformer-vue',
+                  filePath: asset.filePath,
+                },
+              });
+          }
+          if (options.autoinstall && toInstall) {
+            await options.packageManager.require(toInstall, asset.filePath, {
+              autoinstall: true,
```

Comment:
If you remove `options.autoinstall from the condition` and the value directly to the package manager, an error would be thrown.

I'm guessing `compileStyleAsync` fails as well when the package is missing?

Reply:
Yes. If you'd like I could do what you are suggesting instead, since a Parcel diagnostic would be thrown with your suggestion as opposed to a standard error from `compileStyleAsync`.